job-service-acceptance-tests/pom.xml

```
@@ -625,6 +625,10 @@
                                     <CAF_RABBITMQ_PASSWORD>guest</CAF_RABBITMQ_PASSWORD>
                                     <CAF_SCHEDULED_EXECUTOR_PERIOD>2</CAF_SCHEDULED_EXECUTOR_PERIOD>
                                     <CAF_WORKER_INPUT_QUEUE>jobservicescheduler-in</CAF_WORKER_INPUT_QUEUE>
+                                    <!-- Can't use quorum as the integration tests (the ones that are not disabled) are
+                                    using an old worker that does not support the 'queueType' prop in its js config:
+                                    cfg_test_worker-batch_RabbitWorkerQueueConfiguration -->
+                                    <CAF_RABBITMQ_QUEUE_TYPE>classic</CAF_RABBITMQ_QUEUE_TYPE>
```

How is this only being added now?  Were the tests silently failing before?

---

They were passing before because the job status was not being changed to 'Failed' when it failed with a non-trasnient error, and now it does, so these assertions started failing:

```
[ERROR]   JobServiceEndToEndIT.testPausePausedJobIsSuccessful:1260->pauseJob:1531 ▒ Api {"message":"ERROR: job_id {J1754557081903-1911} cannot be paused as it has a status of {Failed}. Only jobs with a status of Active or Waiting can be paused.\n  Where: PL/pgSQL function pause_job(character varying,character varying) line 23 at RAISE"}
[ERROR]   JobServiceEndToEndIT.testPauseWaitingJobIsSuccessful:1251->pauseJob:1531 ▒ Api {"message":"ERROR: job_id {J1754557082073-7842} cannot be paused as it has a status of {Failed}. Only jobs with a status of Active or Waiting can be paused.\n  Where: PL/pgSQL function pause_job(character varying,character varying) line 23 at RAISE"}
[ERROR]   JobServiceEndToEndIT.testResumeCancelledJobIsNotAllowed:1406->cancelJob:1523 ▒ Api {"message":"ERROR: job_id {J1754557082222-4273} cannot be cancelled\n  Where: PL/pgSQL function cancel_job(character varying,character varying) line 24 at RAISE"}
[ERROR]   JobServiceEndToEndIT.testResumePausedJobThatWasPreviouslyWaitingMovesJobToActiveStatus:1310->pauseJob:1531 ▒ Api {"message":"ERROR: job_id {J1754557082493-6876} cannot be paused as it has a status of {Failed}. Only jobs with a status of Active or Waiting can be paused.\n  Where: PL/pgSQL function pause_job(character varying,character varying) line 23
```